vmod/vmod_std.vcc

```diff
@@ -582,6 +582,10 @@ $Function VOID rollback(HTTP h)
 
 Restores the *h* HTTP headers to their original state.
 
+.. note: after calling this function, you must immediately either `return(restart)` or `return(retry)`.
```

Comment:
I think this is what you are looking for:
```suggestion
.. note:: after calling this function, you must immediately either `return(restart)` or `return(retry)`.
```

The current version is treated as a comment and is not being rendered in the generated man page:
```
man -l man/vmod_std.3 | grep -B 1 -A 6 Restores
   VOID rollback(HTTP h)
       Restores the h HTTP headers to their original state.

       Failure in doing so may result in Varnish panicking.

       Example:

          std.rollback(bereq);
```
Also, please fix the indentation of next line so that it is correctly aligned with the NOTE bloc.